content/docs/command-reference/status.md

```diff
@@ -140,12 +140,12 @@ that.
 - `--show-json` - prints the command's output in easily parsable JSON format,
   instead of a human-readable table.
 
-- `-j <number>`, `--jobs <number>` - parallelism level for DVC to retrieve
-  information from remote storage. This only applies when the `--cloud` option
-  is used, or a `--remote` is given. The default value is `4 * cpu_count()`. For
-  SSH remotes, the default is `4`. Note that the default value can be set using
-  the `jobs` config option with `dvc remote modify`. Using more jobs may speed
-  up the operation.
+- `-j <number>`, `--jobs <number>` - parallelism level for DVC to access data
```

Comment:
where does this change come from? is it related to this PR?

Reply:
I took over this PR and realized the --jobs description originally provided used some old language we weren't supposed to have anymore. Then I realized the descriptions aren't very consistent and just made sure they are. I also fixed some errors along the way like mentioning `remote modify` when it doesn't apply.